tutorial/source/vae.ipynb

```diff
@@ -38,7 +38,7 @@
     "\n",
     "$$ p({\\bf x}, {\\bf z}) = \\prod_{i=1}^N p_\\theta({\\bf x}_i | {\\bf z}_i) p({\\bf z}_i)  $$\n",
     "\n",
-    "The fact that $p({\\bf x}, {\\bf z})$ breaks up into a product of terms like this makes it clear what me mean when we call $\\bf z_i$ a local random variable. For any particular $i$, only the single datapoint $\\bf x_i$ depends on $\\bf z_i$. As such the $\\{\\bf z_i\\}$ describe local structure, i.e. structure that is private to each data point. This factorized structure also means that we can do subsampling during the course of learning. As such this sort of model is amenable to the large data setting. (For more discussion on this and related topics see [SVI Part II](svi_part_ii.ipynb).)\n",
+    "The fact that $p({\\bf x}, {\\bf z})$ breaks up into a product of terms like this makes it clear what I mean when we call $\\bf z_i$ a local random variable. For any particular $i$, only the single datapoint $\\bf x_i$ depends on $\\bf z_i$. As such the $\\{\\bf z_i\\}$ describe local structure, i.e. structure that is private to each data point. This factorized structure also means that we can do subsampling during the course of learning. As such this sort of model is amenable to the large data setting. (For more discussion on this and related topics see [SVI Part II](svi_part_ii.ipynb).)\n",
```

Comment:
Thanks for the fix! Can we change `I` --> `we`, since we are using the collective form throughout our tutorials?

Reply:
Hi, thanks for reviewing! Rebased with dev and made the change from "I" to "we"